### DISCUSSION TITLE
Getting to a more phonemic-like transcription after alignment

### DISCUSSION THREAD

**Discussion by praat-enthusiast** [MontrealCorpusTools/mfa-models#29] [Pronunciation dictionaries]
I'm aware that recent versions of MFA IPA dictionaries follow the opinionated phone set laid out [here](https://mfa-models.readthedocs.io/en/latest/mfa_phone_set.html), which produces a more allophonic transcription. However, I'm based in sociolinguistics and for the project I'm currently working on we would be quite interested to end up with more phonemic or broad phonetic transcription (essentially a version with all of the rules described [here](https://mfa-models.readthedocs.io/en/latest/mfa_phone_set.html) reversed).

I was wondering if anyone has a version of the IPA dictionary for UK English which doesn't have the rules described implemented, or has already created a script of some kind to get back to a more standard phonemic transcription after alignment? As I understand it, the current acoustic models have been trained with the dictionaries that use the opinionated phone set, and the allophonic detail in the models and dictionaries improves the alignment, so we would likely be aligning using this phone set and then trying to revert back to a phonemic transcription afterwards. If anyone has attempted this, or has access to a previous version of the dictionary which doesn't implement the new phone set, I'd really appreciate it if you'd be willing to share this with me! I believe that a more phonemic-like dictionary once existed for the US English IPA dictionary at least, as it appears to be mentioned [here](https://memcauliffe.com/bootstrapping-an-ipa-dictionary-for-english-using-montreal-forced-aligner-20.html).

If no one has attempted this already, I'm planning to write a script that will reverse-engineer the rules and produce a more phonemic transcription - I'll share it here if this attempt is successful!

**Comment by thealk** [comment]
Piggy-backing on this because this has also become of interest for our group recently. We'd like to take advantage of the global english acoustic model, but ideally would have continued using the [US English Arpabet dictionary](https://mfa-models.readthedocs.io/en/latest/dictionary/English/English%20%28US%29%20ARPA%20dictionary%20v3_0_0.html#English%20(US)%20ARPA%20dictionary%20v3_0_0). I know that's not possible because they have different phone sets, but if anyone has a workaround I'd also be interested (would also be interested in your solution if you wind up going down that path).

**Comment by jhwormald** [comment]
@praat-enthusiast Wondering if you ever had any luck with this? This is something I'm interested in for a few projects I'm working on with some others.

**Comment by halldamien** [comment]
Also piggybacking here because I'm going to have exactly this question, but about French, not English. I work on the sociolinguistics of regional Frenches. I'll post here as I make progress or decisions, but I'm interested in the progress or decisions that others with similar questions / problems make.
